charts/orchestrator/values.yaml

```diff
@@ -66,7 +66,7 @@ backstage:
   upstream:
     backstage:
       image:
-        tag: next
+        tag: pr-885
```

Comment:
The PR image lives for ~2 weeks before being cleaned-up. Therefore we can't rely on it. However, the rest of this PR still has value.